src/custom-apps/views/CustomAppList.tsx

```diff
@@ -0,0 +1,127 @@
+import { useApolloClient } from "@apollo/client";
+import AppDeleteDialog from "@saleor/apps/components/AppDeleteDialog";
+import { EXTENSION_LIST_QUERY } from "@saleor/apps/queries";
+import { WindowTitle } from "@saleor/components/WindowTitle";
+import {
+  AppsListQuery,
+  AppSortField,
+  AppTypeEnum,
+  OrderDirection,
+  useAppDeleteMutation,
+  useAppsListQuery,
+} from "@saleor/graphql";
+import useNavigator from "@saleor/hooks/useNavigator";
+import useNotifier from "@saleor/hooks/useNotifier";
+import { sectionNames } from "@saleor/intl";
+import createDialogActionHandlers from "@saleor/utils/handlers/dialogActionHandlers";
+import React from "react";
+import { useIntl } from "react-intl";
+
+import CustomAppListPage from "../components/CustomAppListPage";
+import { messages } from "../messages";
+import {
+  customAppListUrl,
+  CustomAppListUrlDialog,
+  CustomAppListUrlQueryParams,
+  customAppUrl,
+} from "../urls";
+
+const getCurrentAppName = (
+  id: string,
+  collection?: AppsListQuery["apps"]["edges"],
```

Comment:
**suggestion:** We probably have a fragment for that, something like AppFragment.

Reply:
I have extracted it into `findById` util.